x-pack/plugin/esql/qa/testFixtures/src/main/resources/match-operator.csv-spec

```diff
@@ -102,6 +102,64 @@ book_no:keyword | title:text
 7140            |The Lord of the Rings Poster Collection: Six Paintings by Alan Lee (No. 1)     
 ;
 
+
+matchWithDisjunction
+required_capability: match_function
```

Comment:
We might need to add a new esql capability if the bwc tests fail 🤔

Reply:
I was too optimistic...

![image](https://github.com/user-attachments/assets/ab573331-9f4f-46c3-bbfc-e785650b6840)

Reply:
I love this ! 🤟